src/bitcoin-cli.cpp

```diff
@@ -544,7 +544,9 @@ static int CommandLineRPC(int argc, char *argv[])
             args.erase(args.begin()); // Remove trailing method name from arguments vector
         }
         Optional<std::string> wallet_name{};
-        if (gArgs.IsArgSet("-rpcwallet")) wallet_name = gArgs.GetArg("-rpcwallet", "");
+        if (!gArgs.GetArgs("-rpcwallet").empty()) {
```

Comment:
What is this supposed to improve?

Reply:
This is how `-rpcwallet` was parsed before (see diff in 7430775). I don't think it changes anything, but I mistakenly thought the same with respect to `-getinfo` before your feedback in that PR.

Reply:
That said, I added `-rpcwallet` test coverage recently in #18724, so I think either way works.

Reply:
Reverted to minimal change.